sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryAndroid.kt

```diff
@@ -6,10 +6,6 @@ import io.sentry.android.core.SentryAndroid
 import io.sentry.android.core.SentryAndroidOptions
 
 internal actual object SentryBridge {
```

Comment:
Since you are modifying the Bridge, the `SentryJvm.kt` and `SentryJs.kt` should also fix at least the method signatures.